core/services/chainlink/testdata/config-full.toml

```diff
@@ -179,49 +177,48 @@ Release = 'v1.2.3'
 [[EVM]]
 ChainID = '1'
 Enabled = false
-BalanceMonitorEnabled = true
 BlockBackfillDepth = 100
 BlockBackfillSkip = true
 ChainType = 'Optimism'
-EIP1559DynamicFees = true
 FinalityDepth = 42
 FlagsContractAddress = '0xae4E781a6218A8031764928E88d457937A954fC3'
-GasBumpPercent = 10
-GasBumpThreshold = '6'
-GasBumpTxDepth = 6
-GasBumpWei = '100'
-GasEstimatorMode = 'L2Suggested'
-GasFeeCapDefault = '9223372036854775807'
-GasLimitDefault = '12'
-GasLimitMultiplier = '1.234'
-GasLimitTransfer = '100'
-GasPriceDefault = '9223372036854775807'
-GasTipCapDefault = '2'
-GasTipCapMinimum = '1'
 LinkContractAddress = '0x538aAaB4ea120b2bC2fe5D296852D948F07D849e'
 LogBackfillBatchSize = 17
 LogPollInterval = '1m0s'
-MaxGasPriceWei = '281474976710655'
 MaxInFlightTransactions = 19
 MaxQueuedTransactions = 99
-MinGasPriceWei = '13'
 MinIncomingConfirmations = 13
-MinimumContractPayment = '9223372036854775807'
+MinimumContractPayment = '9.223372036854775807 link'
 NonceAutoSync = true
-OCRContractConfirmations = 11
-OCRContractTransmitterTransmitTimeout = '1m0s'
-OCRDatabaseTimeout = '1s'
-OCRObservationTimeout = '1s'
-OCRObservationGracePeriod = '1s'
-OCR2ContractConfirmations = 7
 OperatorFactoryAddress = '0xa5B85635Be42F21f94F28034B7DA440EeFF0F418'
 RPCDefaultBatchSize = 17
 TxReaperInterval = '1m0s'
 TxReaperThreshold = '1m0s'
 TxResendAfterThreshold = '1h0m0s'
 UseForwarders = true
 
-[EVM.BlockHistoryEstimator]
+[EVM.BalanceMonitor]
+Enabled = true
+BlockDelay = 17
+
+[EVM.GasEstimator]
+Mode = 'L2Suggested'
+PriceDefault = '9.223372036854775807 ether'
+PriceMaxWei = '281.474976710655 micro'
+PriceMinWei = '13'
+LimitDefault = '12'
+LimitMultiplier = '1.234'
+LimitTransfer = '100'
+BumpPercent = 10
+BumpThreshold = '6'
+BumpTxDepth = 6
+BumpWei = '100'
+EIP1559DynamicFees = true
+FeeCapDefault = '9.223372036854775807 ether'
+TipCapDefault = '2'
+TipCapMinimum = '1'
+
+[EVM.GasEstimator.BlockHistory]
```

Comment:
@samsondav WDYT about this `[GasEstimator]` grouping? I started with just `[Gas]`, but if 'estimator' is not confusing or inaccurate, then it makes things a little simpler overall. I don't feel strongly about the particular names, but I do think that grouping these and standardizing the names brings some clarity. We could also go a little further by aligning the relevant field names to either end in `*Wei`, or drop it and default/require to including it on the value:
```diff
PriceDefault = '9.223372036854775807 ether'
-PriceMaxWei = '281.474976710655 micro'
+PriceMax = '281.474976710655 micro'
-PriceMinWei = '13'
+PriceMin = '13 wei'
```

Reply:
Did this, also switched the gas _amount_ fields from strings to primitive types since they are much easier to work with (e.g. `21_000`, `1_000_000`) and are little to no risk of overflow, since `uint32` is already more than double precision of a whole block's gas limit.